packages/assistant/src/RPA/Assistant/background_flet.py

```diff
@@ -12,6 +13,7 @@ class BackgroundFlet:
     """Class that manages the graphical flet subrocess and related operations"""
 
     def __init__(self):
+        atexit.register(self.close_flet_view)
```

Comment:
this seems to be missing from the described changes, is this a regression (aka something we've lost along the way) or is it something we thought it's not needed when it actually is?

Reply:
Yeah. If running through pure python and using the red-x on macOS to quite the application, the flet UI stays around. 

This fixes that.

Reply:
Documented to releasenotes.